pkg/port/builtin/parent/parent.go

```diff
@@ -140,8 +141,13 @@ func (d *driver) AddPort(ctx context.Context, spec port.Spec) (*port.Status, err
 	}
 	routineStopCh := make(chan struct{})
 	routineStop := func() error {
-		close(routineStopCh)
-		return nil // FIXME
+		routineStopCh <- struct{}{}
```

Comment:
This can be delivered to the `copyConnToChild(c, socketPath, spec, stopCh)` routine in `tcp.go`, so this does not work as expected

Reply:
Opened PR https://github.com/rootless-containers/rootlesskit/pull/265 to  fix this